plugins/dapper/README.md

```diff
@@ -0,0 +1,168 @@
+
+   # Install specific Linux datasets
+   cargo run -- . --install ubuntu-focal
+   cargo run -- . --install ubuntu-jammy
+   cargo run -- . --install ubuntu-noble
+   cargo run -- . --install debian-bookworm
+
+   # Install all available datasets
+   cargo run -- . --install all
+   ```
+
+## Installation
+
+In the same virtual environment that Surfactant was installed in, install this plugin:
+
+```bash
+# From PyPI (when available)
+pip install surfactantplugin-dapper
+
+# From GitHub
+pip install git+https://github.com/LLNL/Surfactant#subdirectory=plugins/dapper
+
+# For developers making changes to this plugin
+git clone https://github.com/LLNL/Surfactant.git
+cd Surfactant/plugins/dapper
+pip install -e .
+```
+
+
+## Output Format
+
+The plugin adds package information to the metadata field of software entries in the SBOM. For each binary file, it provides:
+
+```json
+{
+  "dapper_packages": [
+    {
+      "package_name": "coreutils",
+      "full_package_name": "coreutils/focal 8.30-3ubuntu2",
+      "package_dataset": "ubuntu-focal",
+      "normalized_name": "cat",
+      "original_name": "cat",
+      "file_path": "bin/cat"
+    },
+    {
+      "package_name": "coreutils",
+      "full_package_name": "coreutils/jammy 8.32-4.1ubuntu1",
+      "package_dataset": "ubuntu-jammy",
+      "normalized_name": "cat",
+      "original_name": "cat",
+      "file_path": "bin/cat"
+    }
+  ]
+}
+```
+
+### Key Fields
+
+- **package_name**: Short package name (e.g., "libssl3")
+- **full_package_name**: Complete package identifier with version
+- **package_dataset**: Source dataset/distribution
+- **normalized_name**: Normalized filename used for matching
+- **original_name**: Original filename as found
+- **file_path**: Installation path within the package
+
+## Configuration
+
+### Enabling/Disabling
+
+The plugin can be controlled using Surfactant's plugin management features with the plugin name `surfactantplugin_dapper` (defined in `pyproject.toml`).
+
+```bash
+# Disable the plugin
+surfactant plugin disable surfactantplugin_dapper
+
+# Enable the plugin
+surfactant plugin enable surfactantplugin_dapper
+```
+
+### Dataset Management
+
+All dataset commands must be run from the dapper directory:
+
+```bash
+cd ~/dapper
+```
+
+Then run:
+
+```bash
+# List installed datasets
+cargo run -- . --list-datasets
+
+# Update all datasets
+cargo run -- . --update all
+
+# Remove a dataset
+cargo run -- . --uninstall ubuntu-focal
+```
+
+## Supported File Types
+
+Currently supported:
+- **ELF files** (Linux binaries and libraries): `.so`, `.o`, and extensionless executables
+
+Planned support:
+- **PE files** (Windows binaries): `.dll`, `.exe`, `.sys` (pending NuGet dataset availability)
+
+
+## Uninstalling
+
+Remove the plugin with:
+```bash
+pip uninstall surfactantplugin-dapper
+```
+
+If pipx was used:
+```bash
+pipx uninject surfactant surfactantplugin-dapper
+```
+
+## License
+
+MIT License (same as Surfactant)
+
+## Additional Resources
+
```

Comment:
https://crates.io/crates/dapper can be added to the list of resources for dapper.